_config.yml

```diff
@@ -119,14 +119,10 @@ exclude:
 plugins:
   - jekyll-paginate
   - jekyll-redirect-from
-  - jekyll-get-json
+#   - jekyll-get-json
 
 ## external data sources
```

Comment:
Perhaps make a note here that the remote JSON files have been replaced with the local R scripts?

Reply:
Done in https://github.com/hpc-carpentry/hpc-carpentry.github.io/pull/25/commits/59c6fd76f582e98d9ddf1d91020559401b972b56